topics/twrp/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-created_by: Team Win
+created_by: TeamWin
 display_name: TWRP (software)
 short_description: Team Win Recovery Project (TWRP), pronounced "twerp", is an open-source software custom recovery image for Android-based devices.
```

Comment:
Should we update here too?

Reply:
What do you mean?

Reply:
```suggestion
short_description: TeamWin Recovery Project (TWRP), pronounced "twerp", is an open-source software custom recovery image for Android-based devices.
```

Reply:
Although I see on https://twrp.me/about/ that they refer to themselves as `Team Win`. So I'm not sure either of these changes are necessary.

Reply:
Yes, okay. Is there no way to redirect the 'Team Win' to the 'TeamWin' github profile?

Reply:
Ok, if that’s the case you can do this:
```suggestion
short_description: Team Win Recovery Project (TWRP), pronounced "twerp", is an open-source software custom recovery image for Android-based devices.
github_url: https://github.com/TeamWin
```